CHANGELOG.adoc

```diff
@@ -51,6 +51,7 @@ The use of generics has thus been greatly reduced since they could not create a
 - https://github.com/eclipse-sirius/sirius-web/issues/3279[#3279] [core] Remove all references to the Stereotype API from `sirius-components-core` since it has always been a Sirius Web API.
 This will make the dependency to the old Sirius Web architecture more explicit.
 - https://github.com/eclipse-sirius/sirius-web/issues/3279[#3279] [sirius-web] Simplify the lifecycle and API of the documentation creation in order to provide a simpler yet more powerful API for the new architecture
+- https://github.com/eclipse-sirius/sirius-web/issues/3382[#3382] [diagram] Remove the hardcoded show/hide tools in the node and edge palettes.
```

Comment:
I updated the changelog to better reflect the changes.
I also updated Flow and Papaya to add the new hide/show tool sections. I didn't know how to update domain since it is derived from the `domain.odesign` file. I see two options to do it:
- Update the `domain.odesign` file with a fake show/hide tool. It doesn't make much sense because these tools aren't required in Sirius but I guess we can do it
- Update `CompatibilityPaletteProvider#createExtraToolSections` to add the hide/show tools as extra tools (the same way direct edit and delete are added right now). This would make more sense but since we want to remove hardcoded tools in the "normal" layer, I don't know if we want to keep them in the compatibility layer.

Reply:
Ok, after a discussion with @sbegaudeau, we have decided to not update domain.odesign.
But, domain has been migrated to View DSL in the architecture, so it means you should be able to add the hide tools in this part of the code.